docs/use-cases/deploy-ml.md

```diff
@@ -1,77 +1,113 @@
+For an object detector, use bounding boxes.
 
 {{< /tablestep >}}
 {{< tablestep >}}
-{{<imgproc src="/services/ml/label.svg" class="fill alignleft" style="max-width: 300px" declaredimensions=true alt="Label data">}}
-**2. Create a dataset and label**
+{{<imgproc src="/services/ml/label.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Label data">}}
+**2. Create a dataset [_(i)_](/services/data/dataset/)**
 
-Once you have enough images of the objects you'd like to classify, [label your data and create a dataset](/services/data/dataset/) in preparation for training classification models.
+Use the interface on the **DATA** tab (or the [`viam data dataset add` command](/cli/#data)) to add all images you want to train the model on to a dataset.
 
 {{< /tablestep >}}
+{{< /table >}}
+
+## Train and test a machine learning (ML) model
+
+{{< table >}}
 {{< tablestep >}}
-{{<imgproc src="/services/ml/train.svg" class="fill alignright" style="max-width: 300px" declaredimensions=true alt="Train models">}}
-**3. Train an ML model**
+{{<imgproc src="/services/ml/train.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Train models">}}
+**1. Train an ML model [_(i)_](/services/ml/train-model/)**
 
-Use your labeled data to [train your own models](/services/ml/train-model/) for object classification using data from the [data management service](/services/data/).
+In the Viam app, navigate to your list of [**DATASETS**](https://app.viam.com/services/data/datasets) and select the one you want to train on.
+Click **Train model** and follow the prompts.
 
 {{< /tablestep >}}
 {{< tablestep >}}
-{{<imgproc src="/registry/upload-module.svg" class="fill alignleft" style="max-width: 200px" declaredimensions=true alt="Train models">}}
-**4. Deploy your ML model**
+{{<imgproc src="/registry/upload-module.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Train models">}}
+**2. Deploy your ML model**
 
-To use ML models with your machine, use the built-in [ML model service](/services/ml/) to deploy and run the model.
+Configure the built-in [ML model service](/services/ml/deploy/) to deploy and run the model.
+Once you've added the ML model service to your machine, choose your newly-trained model from the dropdown menu in the ML model service's configuration card.
 
 {{< /tablestep >}}
 {{< tablestep >}}
-{{<imgproc src="/services/ml/configure.svg" class="fill alignright" style="max-width: 300px" declaredimensions=true alt="Configure a service">}}
-**5. Configure an <code>mlmodel</code> vision service**
+{{<imgproc src="/services/icons/vision.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Configure a service">}}
+**3. Configure an <code>mlmodel</code> vision service [_(i)_](/services/vision/)**
+
+You can think of the vision service as the bridge between the ML model service and the output from your camera.
```

Comment:
I don't think a bridge adds enough meaning as a metaphor here - so adding a bit extra
```suggestion
You can think of the vision service as the bridge between the ML model service and the output from your camera.
The vision service applies the ML model to the images stream from the camera.
```

Reply:
I can just get rid of it
```suggestion
The vision service takes the the ML model and applies it to the stream of images from your camera.
```